### PR TITLE
Problem: These deleted functions are defined lower in this function. …

### PR DESCRIPTION
**Pull request by kf6nux**
… This will cause confusion if/when a developer later tries to alter the behavior of one of these functions.

Solution: Delete the unused definitions.

### CODE REVIEW

File: src/scripts/pagerduty.coffee [hunk 1]
```diff
@@ -690,69 +690,6 @@ module.exports = (robot) ->
     match.split(/[ ,]+/).map (incidentNumber) ->
       parseInt(incidentNumber)
 
-  userEmail = (user) ->
-    user.pagerdutyEmail || user.email_address || user.profile?.email || process.env.HUBOT_PAGERDUTY_TEST_EMAIL
-
-  campfireUserToPagerDutyUser = (msg, user, required, cb) ->
```
Comment (stephenyeargin) [inline src/scripts/pagerduty.coffee]: There appears to still be calls to this method in particular on [L758](https://github.com/hubot-scripts/hubot-pager-me/pull/115/files#diff-b6367b6561a9b2675121a45cd273efceL758).

Comment (kf6nux) [reply src/scripts/pagerduty.coffee]: It's still defined here: https://github.com/hubot-scripts/hubot-pager-me/blob/1a2093c71489b7546b80ebe2f058b1fe6d071e90/src/scripts/pagerduty.coffee#L869

Comment (stephenyeargin) [reply src/scripts/pagerduty.coffee]: :+1:

**Comment by stephenyeargin** [comment]
Released with [`v3.1.0`](https://github.com/hubot-scripts/hubot-pager-me/releases/tag/v3.1.0)

**Pull request merged**